lib/decoder_impl.cc

```diff
@@ -770,7 +769,7 @@ namespace gr {
                                                        this->d_samples_per_symbol / this->d_corr_decim_factor,
                                                        &index_correction);
 
-                        if (c > 0.8f) {
+                        if (c > 0.9f) {
```

Comment:
Due to a missing factor 2 in the function call to `sliding_cross_correlate_upchirp`, this was indeed wrong. However, the new algorithm demonstrates much higher accuracy results with 0.9.